charts/alfresco-search-enterprise/docs/truststore.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Truststore
+parent: Alfresco Search Enterprise
+grand_parent: Guides
+---
+
+# Setting custom java truststore
+
+There multiple use cases where you may want to set a custom keystore or
+truststore. It can be because you need to interact with a third party service
+which uses a self-signed certificate or a prvite CA that is not shipped with
+the container. Or maybe the service you are trying to connect to uses a
+self-signed certificate and you want to trust it. In this case, you can
+create a custom truststore and set it in the container.
+
+## Create a custom truststore
+
+You can create a custom truststore on your local machine using the
+`keytool` command. The `keytool` command is part of the Java Development Kit (JDK)
+and is used to manage keystores and truststores. You can use the following
+command to create a new truststore and import a certificate into it:
+
+```bash
+keytool -importcert -file <path-to-certificate-file> -alias <alias-name> -keystore <path-to-truststore-file>
+```
+
+Where `<path-to-certificate-file>` is the path to the CA certificate or the
+self-signed certificate you want to trust, `<alias-name>` is a name you want
+to give to the certificate in the truststore, and `<path-to-truststore-file>`
+is the path to the truststore file you want to create.
+You will be prompted to enter a password for the truststore. You can use
+the default password `changeit` or leave it empty (Truststore usually contain
+public informations so it should not be a problem to not use a password).
+Also note that <path-to-truststore-file> can point to an already existing
+truststore file. We recommand using the default  `cacerts` truststore that
+comes with the JDK. This way, you can add your custom certificates to the
+default truststore and use it in your application.
```

Comment:
isn't this something we are telling over and over again - at a different degree depending on how much we are into describing custom trust store creation?

maybe you can extract this in a dedicated page, maybe still in guides which just describe keystore handling (if really there isn't a better piece of external/official documentation which describe this)

Reply:
This is what I started doing but unfortunately it quickly appeared to me this chart is kind of special due to the iterative structure of the values file (liveIndexing.*.*) and the merging mechanism that exists for environment variable. The repo also have a similar page that do not work EXACTLY the same but is close.